muxpi/dut.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+echo "Setting up DUT" 
+stm_armv7 -clr
+stm_armv7 -print "DUT"
+stm_armv7 -dut
+echo "This script contains a lot of sleep commands as it needs to wait for the screens. Patience is key!"
+sleep 30s
+
+stty -F /dev/ttyACM0 115200 cs8 -cstopb -parenb
+cd /home/muxpi/
+#We need one screen to save the output of /dev/ttyACM0. Everything will be saved on 'serial-output'
+screen -d -m -S output 
+screen -S output -X stuff 'cat /dev/ttyACM0 > serial-output\n' 
+sleep 3s
+
+#This screen will be logging in the rpi and sending the commands to it. Note that when logging in, '\n' has to be passed multiple times as the login might fail. 
+screen -d -m -S input /dev/ttyACM0 115200 
+screen -S input -X stuff '\nroot\n\n\n'
+sleep 5s
+#This returns either degraded or running. If it doesnt return either one, then something is wrong and rpi did not boot. This can be later grepped from `serial-output`
+screen -S input -X stuff '\n\n\n systemctl is-system-running --wait\n'
+
+sleep 5s
+
+#Kill both of the screens as our job is done. 
+screen -X -S input kill
+screen -X -S output kill
+
+
+sleep 5s
+stm_armv7 -ts 
+stm_armv7 -clr
+stm_armv7 -print "Trigger me!"
```

Comment:
You have added more comments for the self-explanatory parts, than for the parts which are not self-explanatory. Please add some more comments to the sections with stm_armv7 commands with "magic arguments".

Reply:
I mean that the comments can be grouped together in sections or parts of the script. You do not need one comment per command, but just a summary of the purpose of the sections, so that it's easier to follow along when reading the script.

Reply:
Some additional logging would be nice, so it's clear what's happening

Reply:
I feel like there are enough logging on set-up-image.sh (which calls dut.sh). On DUT there is a lot of screens happening and even if the connection is not made, the screens will not be failing and the log output might distract you from the real problem.